torch/fx/immutable_collections.py

```diff
@@ -0,0 +1,13 @@
+def _no_mutation(self, *args, **kwargs):
+    raise NotImplementedError(f"'{type(self).__name__}' object does not support mutation")
```

Comment:
Can we tell the user to explicitly assign new `args` and `kwargs`? Just so it's obvious what do you if you've gotten here